src/shared/extensionUtilities.ts

```diff
@@ -80,12 +86,29 @@ export function getIdeProperties(): IdeProperties {
                 return createCloud9Properties(localize('AWS.title.cn', 'Amazon'))
             }
             return createCloud9Properties(company)
+        case IDE.sagemaker:
+            if (isCn()) {
+                // check for cn region
+                return createSageMakerProperties(localize('AWS.title.cn', 'Amazon'))
+            }
+            return createSageMakerProperties(company)
         // default is IDE.vscode
         default:
             return vscodeVals
     }
 }
 
+function createSageMakerProperties(company: string): IdeProperties {
+    return {
+        shortName: localize('AWS.vscode.shortName', 'AWS Code'),
```

Comment:
```suggestion
        shortName: localize('AWS.vscode.shortName', '{0} Code', company),
```

Reply:
`company` is "Amazon" here, so this would result in a different short name then desired